app/tasks/mailchimp.py

```diff
@@ -24,8 +27,8 @@ def get_client():
     headers = default_headers()
     headers['User-Agent'] = 'Digital Marketplace (marketplace.service.gov.au)'
     client = MailChimp(
-        getenv('MAILCHIMP_USERNAME'),
         getenv('MAILCHIMP_SECRET_API_KEY'),
+        getenv('MAILCHIMP_USERNAME'),
```

Comment:
Arguments have been reversed to accomodate the upgrade of the Mailchimp client.  See https://github.com/charlesthk/python-mailchimp#upgrading-from-v2x